projects/libaom/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+MAINTAINER urvang@google.com
+RUN apt-get update && apt-get install -y cmake yasm wget
+RUN git clone https://aomedia.googlesource.com/aom
+RUN wget -q https://storage.googleapis.com/aom-test-data/fuzzer/dec_fuzzer_seed_corpus.zip
```

Comment:
I took a look at this seed corpus. It is pretty large. Is it minimized? (you can do this with libFuzzer with this command: ./fuzzer $EMPTY_CORPUS $INITIAL_CORPUS -merge=1).

This doesn't block this PR since ClusterFuzz can handle this. But ideally the seed corpus should be minimized and only containing small testcases (<1 MB, otherwise there's a lower chance for the fuzzer to mutate important bytes).

Reply:
Thanks for the suggestion!
Currently, we have been using this seed corpus for some other testing, so wanted to start with the same corpus.
Later, we can minimize this as you suggested.

Reply:
No problem.